service/service.go

```diff
@@ -0,0 +1,53 @@
+package service
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	goaclient "github.com/goadesign/goa/client"
+
+	"github.com/fabric8-services/fabric8-common/errors"
+	"github.com/fabric8-services/fabric8-common/goasupport"
+
+	clusterclient "github.com/fabric8-services/fabric8-cluster-client/cluster"
+)
+
+type Service interface {
+	UserClusters(ctx context.Context) (*clusterclient.ClusterList, error)
+}
+
+func NewClusterService(clusterURL string) (Service, error) {
+	url, err := url.Parse(clusterURL)
+	if err != nil {
+		return nil, err
+	}
+	return &service{url}, nil
+}
+
+type service struct {
```

Comment:
maybe name this to `serviceImpl` to avoid the confusion with the `Service` interface ?

Reply:
I will go with `service`.